src/target/stm32f1.c

```diff
@@ -272,13 +276,16 @@ void mm32l0_mem_write_sized(adiv5_access_port_s *ap, uint32_t dest, const void *
 		case ALIGN_BYTE: {
 			uint8_t value;
 			memcpy(&value, src, sizeof(value));
-			tmp = (uint32_t)value << (8U * (dest & 3U));
+			tmp = (uint32_t)value;
```

Comment:
This makes us wonder if multiplying by 0x01010101 might be faster here - either way is about as obscure, so a comment talking through what this is doing (and perhaps why) would be good. Nothing complicated, something like "Copy the value to write to all 4 bytes of the uint32_t"

Reply:
Multiplying by 0x01010101: nice.
Kept current code, added comment.

adiv5_mem_write_sized() is used to write stm32 flash, but is broken on mm32 cortex-m0.
mm32l0_mem_write_sized() is like adiv5_mem_write_sized() but now works on mm32 and stm32.
mm32l0_mem_write_sized() working on mm32 and stm32 means that if future mm32 cortex-m0 behave like an stm32, things work out of the box.